### PR TITLE
create header, footer , adaptive menu

### PR DESCRIPTION
**Pull request by Vitalia888-170**

### CODE REVIEW

**Comment by Shabaldas** [comment]
plz. Fix indents in html and remove .env file from your PR

File: app/javascript/packs/app.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+console.log('js loaded');
```
Comment (Shabaldas) [inline app/javascript/packs/app.js]: remove

File: app/javascript/packs/app.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+			sidebar.classList.toggle("-translate-x-full");
+			document.body.style.overflow = "hidden";
+		} else if (e.target.closest(".close-btn")) {
+			sidebar.classList.toggle("-translate-x-full");
+			document.body.style.overflow = "visible";
+		} else if (e.target.closest(".dropdown-menu-button")) {
+			dropdownContent.classList.toggle("hidden")
+		}
+	})
+})
```
Comment (Shabaldas) [inline app/javascript/packs/app.js]: add empty space after 17 line

File: app/views/shared/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+				<a href="https://www.instagram.com/3d.storm.des/">
+				<div class="w-5 h-5">
+					<%= image_tag("partials_img/instagram.png") %>
+				</div>
+				</a>
+			</div>
+		</div>
+	</div>
+</footer>
+</div>
```
Comment (Shabaldas) [inline app/views/shared/_footer.html.erb]: add empty space after 45 line

File: app/views/shared/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+			</ul>
+		</nav>
+		</div>
+		<div class="block mobile-menu-button focus:outline-none focus:bg-gray-700 md:hidden">
+			<svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+				<path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+			</svg>
+		</div>
+	</div>
+</header>
```
Comment (Shabaldas) [inline app/views/shared/_header.html.erb]: add empty space after 66 line.
And all idents in this and previous files

File: app/assets/stylesheets/application.css [hunk 5]
```diff
@@ -1,15 +1,10 @@
...
- *= require_tree .
- *= require_self
- */
+@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&family=Inter&display=swap');
+html, body{
+	width:100%;
+	height: 100%;
+}
+body{
+	font-family: 'Inter', sans-serif;
```
Comment (Shabaldas) [inline app/assets/stylesheets/application.css]: we already have this font in Tailwindcss.
 Inside tailwind.config.js

File: tailwind.config.js [hunk 6]
```diff
@@ -18,7 +18,17 @@ module.exports = {
       xl: '1280px',
       '2xl': '1536px',
     },
-    colors: ({ colors }) => ({
+    colors: {
```
Comment (Shabaldas) [inline tailwind.config.js:21]: https://tailwindcss.com/docs/customizing-colors

File: app/javascript/packs/app.js (hunk 1 above)
Comment (Vitalia888-170) [reply app/javascript/packs/app.js]: done

File: app/views/static_pages/home.html.erb [hunk 7]
```diff
@@ -1,40 +1,3 @@
...
-    <h1
-     class="animate-pulse font-bold text-gray-200 text-2xl mb-6">
-    Your TailwindCSS setup is working if this pulses...
-    </h1>
-   </div>
-  </div>
- </div>
+<div class="h-full">
+	<h1>home page</h1>
 </div>
```
Comment (Shabaldas) [inline app/views/static_pages/home.html.erb]: add empty space after 40 line

File: app/views/layouts/application.html.erb [hunk 8]
```diff
@@ -1,18 +1,23 @@
...
+  <%= render 'shared/header' %>
+  <%= yield %>
+  <%= render 'shared/footer' %>
+</body>
 
-  <body>
-    <%= yield %>
-  </body>
-</html>
+</html>
```
Comment (Shabaldas) [inline app/views/layouts/application.html.erb]: add empty space after 23 line

Comment (Shabaldas) [reply app/views/layouts/application.html.erb]: broken idents in this file

File: app/views/layouts/application.html.erb [hunk 9]
```diff
@@ -1,18 +1,23 @@
...
 
-<link rel="stylesheet" href="https://rsms.me/inter/inter.css">
-<%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
+  <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
+  <%= stylesheet_pack_tag 'application' , 'data-turbolinks-track' : 'reload' %>
+  <%= stylesheet_link_tag 'application' , media: 'all' , 'data-turbolinks-track' : 'reload' %>
+  <%= javascript_pack_tag 'app' , 'data-turbolinks-track' : 'reload' %>
```
Comment (Shabaldas) [inline app/views/layouts/application.html.erb]: why do you add it?

File: app/assets/stylesheets/application.css [hunk 10]
```diff
@@ -1,15 +1,10 @@
...
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+@import url('https://fonts.googleapis.com/css2?family=Fredoka+One&family=Inter&display=swap');
```
Comment (AndriiBuhalo) [inline app/assets/stylesheets/application.css]: для чого нам цей лінк на appalication?

Comment (Vitalia888-170) [reply app/assets/stylesheets/application.css]: підключення google fonts

File: app/javascript/packs/app.js (hunk 2 above)
Comment (Vitalia888-170) [reply app/javascript/packs/app.js]: fix

File: app/views/shared/_footer.html.erb (hunk 3 above)
Comment (Vitalia888-170) [reply app/views/shared/_footer.html.erb]: fix

File: app/views/shared/_header.html.erb (hunk 4 above)
Comment (Vitalia888-170) [reply app/views/shared/_header.html.erb]: fix

File: app/assets/stylesheets/application.css (hunk 5 above)
Comment (Vitalia888-170) [reply app/assets/stylesheets/application.css]: fix

File: tailwind.config.js (hunk 6 above)
Comment (Vitalia888-170) [reply tailwind.config.js:21]: thanks, reviewed

File: app/views/static_pages/home.html.erb (hunk 7 above)
Comment (Vitalia888-170) [reply app/views/static_pages/home.html.erb]: fix

File: app/views/layouts/application.html.erb (hunk 8 above)
Comment (Vitalia888-170) [reply app/views/layouts/application.html.erb]: fix

File: app/views/layouts/application.html.erb (hunk 9 above)
Comment (Vitalia888-170) [reply app/views/layouts/application.html.erb]: fix problem

File: app/assets/stylesheets/application.css (hunk 10 above)
Comment (Vitalia888-170) [reply app/assets/stylesheets/application.css]: fixed

**Review by Shabaldas — APPROVED** [review]

**Review by AndriiBuhalo — APPROVED** [review]
👍🏿

**Pull request merged**